translations/translations.json

```diff
@@ -4811,8 +4811,8 @@
       "refresh": "በቅርብ የሚገኙት ኣከባቢዎች መዝገብ ማሻሻል",
       "pageTitle": "ለእርስዎ አካባቢያዊ መረጃ",
       "metaDescription": "{{appName}} በጀርመን ላለው ህይወት የእርስዎ ዲጂታል መመሪያ ነው። አካባቢያዊ መረጃን፣ ክንዉኖችን እና ምክርን ያግኙ። በእርስዎ ቋንቋ ሁልጊዜ የቅርብ ጊዜ የሆነ እና የሚገኝ ነው።",
-      "cityNotFound": "ያንተ ኣከባቢ ኣልተገኘምን?",
-      "suggestToRegion": "እዚህ ተጫን"
+      "cityNotFound": "",
```

Comment:
While the result for translator is probably the same, I think it would be better to just delete the whole properties:
```suggestion
```